modules/ROOT/pages/index.adoc

```diff
@@ -8,26 +15,151 @@
 
 :console: 0
 
-This is the Neo4j Operations Manual, which includes all the operational details and instructions for installing and deploying Neo4j on-premise and in the cloud.
+[.display.cards.selectable]
 
-For all information on *upgrading and migrating Neo4j*, see the link:{neo4j-docs-base-uri}/upgrade-migration-guide/current/[Neo4j Upgrade and Migration Guide].
+== Manage and monitor your deployments
 
-For more information on link:https://neo4j.com/aura/[Aura], the Neo4j fully managed cloud service, see the link:{neo4j-docs-base-uri}/aura[Neo4j Aura Manual].
+Find here details and instructions for installing and deploying Neo4j on-premise and in the cloud.
+Follow tutorials and find administrative commands to help you manage and monitor your projects.
 
+=== Installation
 
-The latest version of Neo4j is *{neo4j-version-exact}*.
+[.icon]
+image:icon-import.svg[]
 
-ifdef::backend-pdf[]
-include::license.adoc[leveloffset=+1]
-endif::[]
+[.description]
+Install Neo4j in different deployment contexts, such as Linux, macOS, and Windows.
 
-ifdef::backend-html5[(C) {copyright}]
-ifndef::backend-pdf[]
+[.link]
+xref:installation/index.adoc[]
 
-Documentation license: link:{common-license-page-uri}[Creative Commons 4.0]
-endif::[]
-ifdef::backend-pdf[]
-(C) {copyright}
+[.cards]
+=== Upgrade and migration
 
-Documentation license: <<license, Creative Commons 4.0>>
-endif::[]
\ No newline at end of file
+[.icon]
+image:ROOT:icon-upgrade.svg[]
+
+[.description]
+See information on how to upgrade or migrate self-managed Neo4j deployments.
+
+[.link]
+xref:installation/index.adoc[]
+
+=== Aura instances
+
+[.icon]
+image:ROOT:icon-aura.svg[]
+
+[.description]
+See administrative options for fully managed cloud deployment.
+
+[.link]
+xref:installation/index.adoc[]
```

Comment:
This link is wrong because linking to another docset (using link instead of xref) is causing a problem in the UI, so I will fix that later if you agree with the suggestion of the content.